apps/condo/domains/miniapp/gql.js

```diff
@@ -6,6 +6,10 @@
 
 const { gql } = require('graphql-tag')
 
+const { generateGqlQueries } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+
+const COMMON_FIELDS = 'id dv sender { dv fingerprint } v deletedAt newId createdBy { id name } updatedBy { id name } createdAt updatedAt'
```

Comment:
can we move common fields to common domain?

Reply:
Everywhere 🥇

Reply:
Yes, we can....But is it about this task?